test/test_spriting.js

```diff
@@ -13,9 +13,9 @@ describe("spriting module", function () {
 
   it("gets sprite map data using app assets", function (done) {
     var input = "@import 'assets'; @import 'spriting'; " +
-                ".sprite-map-test { foo: sprite-map('test-sprite-map', sprite-layout(horizontal, " +
-                "(spacing: 5px, alignment: bottom)), 'images/*'); }";
-    var expected = ".sprite-map-test {\n  foo: (sprite-map: true, name: test-sprite-map, sources:" +
+                "/* #{inspect(sprite-map('test-sprite-map', sprite-layout(horizontal, " +
```

Comment:
You can revert these changes and rebase. I merged my PR (#17) that fixes these test (which I forgot I had created).